dist/api_data.py

```diff
@@ -1137,6 +1137,12 @@ def __ge__(self, other):
     wiredtiger_open_log_configuration +\
     wiredtiger_open_tiered_storage_configuration +\
     wiredtiger_open_statistics_log_configuration + [
+    Config('backup_partial_restore', '', r'''
+        When set to true, allow a partial restore on the current database using the
+        WiredTiger metadata backup file. WiredTiger will reconstruct the
+        database files that are only present in the directory and remove all
+        table entries that are not in the directory anymore''', 
+        type='list'),
```

Comment:
The description is no longer correct. It is not set to `true`. I suggest changing the name to be consistent with the backup target naming and call it `backup_restore_target`. The description could be something like:
If non-empty and restoring from a backup, restore only the table object targets listed. WiredTiger will remove all table entries that are not listed in the list from the reconstructed metadata. All targets must be use table: object URI configuration.